src/common/scrub_types.cc

```diff
@@ -158,12 +158,22 @@ void inconsistent_snapset_wrapper::set_headless()
 
 void inconsistent_snapset_wrapper::set_ss_attr_missing()
```

Comment:
@dzafman could you change the method name accordingly, for example, "set_snapset_missing()"?